ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/transform/PubsubMessageToTemplatedString.java

```diff
@@ -20,7 +20,7 @@
  * <p>All message attributes, plus derived attributes provided by {@link DerivedAttributesMap},
  * are available for substitution.
  *
- * @throws IllegalArgumentException if template is not fully resolved for a given message.
+ * <p>Throws IllegalArgumentException if template is not fully resolved for a given message.
```

Comment:
Why the change here?

Reply:
because `@throws` isn't a valid tag on a class (even if that class inherits from `Function`)